tap_snapchat_ads/streams.py

```diff
@@ -70,8 +70,7 @@
             # Reference: https://developers.snapchat.com/api/docs/?python#get-all-roles-in-organization
             'roles': {
                 'key_properties': ['id'],
-                'replication_method': 'INCREMENTAL',
-                'replication_keys': ['updated_at'],
+                'replication_method': 'FULL_TABLE',
```

Comment:
@harshpatel4crest Made 'Roles' stream as FULL_TABLE stream, as we are not getting 'updated_at' from the API response, right?

Reply:
Yes, adding comment.